distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestSecurity.java

```diff
@@ -589,13 +589,46 @@ public void testBasicFederatedAuth() throws Exception {
               .createManagedService(CSW_FEDERATED_SOURCE_FACTORY_PID, cswProperties)
               .getPid();
 
+      String unavailableOpenSourceId = "unavailableOpenSearchSource";
+      openSearchProperties =
+          getOpenSearchSourceProperties(
+              unavailableOpenSourceId, OPENSEARCH_PATH.getUrl(), getServiceManager());
+      openSearchProperties.put("authenticationType", "basic");
+      openSearchProperties.put("username", "bad");
+      openSearchProperties.put("password", "auth");
+      unavailableOpenSearchPid =
+          getServiceManager()
+              .createManagedService(OPENSEARCH_FACTORY_PID, openSearchProperties)
+              .getPid();
+
+      String unavailableCswSourceId = "Unavailable Csw";
+      cswProperties =
+          getCswSourceProperties(unavailableCswSourceId, CSW_PATH.getUrl(), getServiceManager());
+      cswProperties.put("authenticationType", "basic");
+      cswProperties.put("username", "bad");
+      cswProperties.put("password", "auth");
+      unavailableCswPid =
+          getServiceManager()
+              .createManagedService(CSW_FEDERATED_SOURCE_FACTORY_PID, cswProperties)
+              .getPid();
+
+      // Wait for sources. Note: we wait for the "unavailable" OpenSearch source because it
```

Comment:
Thanks for adding this comment in here. This wouldn't be very intuitive and seems like the availability based on bad creds can vary by source.